src/core/sequence_batch_scheduler.cc

```diff
@@ -762,12 +826,22 @@ SequenceBatchScheduler::SequenceBatch::SchedulerThread(
               const auto& request_provider = slot_payload.request_provider_;
               const auto& request_header = request_provider->RequestHeader();
 
-              // If this is the first payload in a sequence then send
-              // the appropriate sequence start indicator to the
-              // backend.
+              // Set the start, end, and ready control tensors
+              // appropriately...
               if ((request_header.flags() &
+                   (InferRequestHeader::FLAG_SEQUENCE_START |
+                    InferRequestHeader::FLAG_SEQUENCE_END)) ==
+                  (InferRequestHeader::FLAG_SEQUENCE_START |
+                   InferRequestHeader::FLAG_SEQUENCE_END)) {
+                request_provider->SetInputOverride(startend_input_overrides_);
```

Comment:
Is this for the case where the "sequence length" is 1?

Reply:
Yes. Both start and end flag set on the request.